connector_jira/tests/test_import_analytic_line.py

```diff
@@ -108,3 +108,85 @@ def _test_import_worklog(self, expected_project, expected_task):
                 'user_id': self.env.user.id,
             }]
         )
+
```

Comment:
am I wrong or you are not testing if the correct TZ is applied?

Reply:
I do, there are 3 more testcases

Reply:
`test_import_worklog_specific`
`test_import_worklog_user`
and
`test_import_worklog_naive`

Reply:
Only `specific` and `user` actually shifts the date

Reply:
oh yeah, sorry, did not catch the date changing.
TIP: next time add a comment on top of it saying what you expect ;)